pkg/analysis_server/test/services/correction/fix_test.dart

```diff
@@ -6106,6 +6106,66 @@ final a = {};
 ''');
   }
 
+  test_tearOffLambda_function_test1() async {
```

Comment:
@bwilkerson I already compiled my local SDK in my machine and this cases are being well fixed, it looks like the problem is in the test